gradle/README.txt

```diff
@@ -0,0 +1,23 @@
+For more information on the Gradle dependency verification file
+(verification-metadata.xml), see the following page:
+
+Verifying dependencies - Gradle User Guide
+https://docs.gradle.org/current/userguide/dependency_verification.html
+
+When upgrading an external dependency to a newer version, update the
+dependency verification file as follows:
+
+$ gradle --write-verification-metadata sha256 help
```

Comment:
This isn't sufficient for many of the dependencies. Gradle won't try to download external dependencies until the point they are used. For example: the `junit` dependency is downloaded only when running `gradle test`, the icu data dependency is downloaded only when building the sdk with `-PCOMPILE_WEBKIT=true`, the libav media libraries (for Linux) are downloaded only when building the sdk with `-PCOMPILE_MEDIA=true -PBUILD_LIBAV_STUBS=true`, etc.